Update README (#180)
GRChombo/GRChombo: README.md

## Fawcett Tutorial

The following tutorial has been created to be aimed at Masters/early PhD
students with some HPC experience. We also assume that you have an account set

> up on fawcett, and that you are happy with using GitHub. Inevitably it will not be 

Reviewer: I thought we might as well add a link to the fawcett documentation up here (even though you did it below as well) and we should say "familarity with git" rather than "GitHub".
```suggestion
up on [fawcett](https://www.maths.cam.ac.uk/computing/faculty-hpc-system-fawcett), and that you are happy with using `git`. Inevitably it will not be 
```

possible to cover everything, but if anything is unclear please do get in contact.

To run simulations with GRChombo, first we need to get a copy of both the
Chombo and GRChombo repositories.

### Getting Chombo and GRChombo

The GRChombo copy of the Chombo repository can be found

> [here](https://github.com/GRChombo/Chombo). To get a copy on your local
> machine, clone it using a command such as

Reviewer: I don't think we want to talk about local machines as this tutorial is specific to running on fawcett. I think we should leave the line as I wrote it originally.
```suggestion
[here](https://github.com/GRChombo/Chombo). To get a copy, clone it using a
command such as
```

```bash
git clone https://github.com/GRChombo/Chombo.git ~/Chombo
```

> e.g. to clone into your home directory.

Reviewer: ```suggestion
e.g. to clone into your home directory (which we will assume below).
```

Next, clone the GRChombo repository (and checkout this branch) using a command
such as
```bash
git clone -b training/fawcett https://github.com/GRChombo/GRChombo.git ~/GRChombo
```

> [Note that if you already have a copy of GRChombo on your local machine, you do not
> need to clone again and can just checkout the training/fawcett branch.]

Reviewer: Likewise, I think we should remove this as this tutorial is specific to running on fawcett.
```suggestion
```

Author: How about
```suggestion
[Note that if you already have a copy of GRChombo on fawcett, you do not
need to clone again and can just checkout the training/fawcett branch.]
```

Reviewer: For simplicity, I would rather just leave it out because then we need to ensure everyone's clones are up-to-date and if they have cloned to a different directory, they will have to modify all of the commands below.

Author: Ok sure

module load intel/impi/2018.3/intel
module load hdf5-intel/1.10.4
```

> Note that you should also check using ```bash module list``` that no competing

Reviewer: This is just to fix the formatting of the Markdown code block.

````suggestion
Note that you should also check using 
```bash 
module list
```
that no conflicting
````

modules are loaded, as this can cause problems with compilation. If they are,

> you can remove using ```module unload module_name```.

Reviewer: Again, just fixing markdown formatting.

````suggestion
you can remove them using 
```bash
module unload <module_name>
```
````
